documentation/External-Grafana/ReadMe.md

```diff
@@ -0,0 +1,179 @@
+# Design proposal 001 External Grafana Instance Integration
+
+## Summary
+
+Introduce integration to External Grafana Instances with Grafana Operator.
+
+This document contains the complete design required for integrating external Grafana instances such as Amazon Managed Grafana with Grafana Operator.
+This includes design elements to support the following with Grafana operator :
+- Ability to define an external Grafana instance as Grafana source to Grafana operator
+- Ability to create Grafana dashboards on a remote Grafana Instance.
+- Ability to add external cloud data sources to an external Grafana instance.
+- Ability to create alerting on a remote Grafana Instance.
+
+## Info
+
+status: Draft
+
+## Motivation
+
+Cloud providers such Amazon Web Services (AWS) have started providing managed remote grafana services such as Amazon Managed Grafana which decouples the responsibilities of managing a grafana instanaces from ops personas and kubernetes environment. 
+
+Currently Grafana operator has an integration to add Amazon Managed Service for Promethus (AMP) as a data source to Grafana instances hosted in a kubernetes environment. As more customers starting to use external grafana services, expanding the Grafana operator to support remote Grafana instances becomes inevitable. Adding ability to integrate with external grafana services, adding data sources, creating dashboards and alerting on a remote Grafana instances offloads responsibilities of managing a grafana instanaces from ops personas which helps them to focus on developing the features required for their business. This helps the customer teams to move from self managed Grafana instance on their Kubernetes environments to Pay as you go model on Grafana instances provided by providers.
+
+## Out of scope
+
+In the design, integration to remote managed Grafana providers other than Amazon Managed Grafana is out of scope at this point. We may add additional integrations as part of future road map items.
+
+## Verification
```

Comment:
This is specific to amazon and it don't really matter, there is no reason why this design document can't solve all external grafana configs.

So the only thing is
`- Create integration tests to create alerting on a remote Grafana Instance.`

Reply:
Agreed and fixed.